### PR TITLE
GE OMAS - Initial Commit

### PR DESCRIPTION
**Pull request by danielaotelea**
GE OMAS provides API to query from governed assets available in the repository.
In this moment, only the Relational Column, Relational Table and Glossary terms are processed (maybe the REST endpoint should be renamed and the Context object should be more generic).
On the Output topic are send events for Classified Entity and New Relationship (Semantic Assignment).

Signed-off-by: Daniela Otelea <daniela-valentina.otelea@ing.com>

### CODE REVIEW

File: open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/objects/Context.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+package org.odpi.openmetadata.accessservices.governanceengine.api.objects;
```
Comment (mandy-chessell) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/objects/Context.java]: Copyright statement

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/processor/ContextBuilder.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+package org.odpi.openmetadata.accessservices.governanceengine.server.processor;
```
Comment (mandy-chessell) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/processor/ContextBuilder.java]: Copyright statement needed

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/publisher/GovernanceEnginePublisher.java [hunk 3]
```diff
@@ -1,18 +1,212 @@
...
+
+    @Override
+    public void processRestoredEntityEvent(String sourceName, String originatorMetadataCollectionId, String originatorServerName, String originatorServerType, String originatorOrganizationName, EntityDetail entity) {
+
+    }
+
+    @Override
+    public void processReTypedEntityEvent(String sourceName, String originatorMetadataCollectionId, String originatorServerName, String originatorServerType, String originatorOrganizationName, TypeDefSummary originalTypeDef, EntityDetail entity) {
+
+    }
```
Comment (mandy-chessell) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/publisher/GovernanceEnginePublisher.java]: Adding unnecessary overrides may prevent default action in the superclass.

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/util/ExceptionHandler.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+package org.odpi.openmetadata.accessservices.governanceengine.server.util;
```
Comment (mandy-chessell) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/util/ExceptionHandler.java]: Copyright statement needed

File: open-metadata-implementation/access-services/governance-engine/governance-engine-spring/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/spring/GovernanceEngineOMASResource.java [hunk 5]
```diff
@@ -129,7 +133,7 @@ GovernedAssetListAPIResponse getGovernedAssets(@PathVariable String serverName,
     @RequestMapping(method = RequestMethod.GET, path = "/assets/{assetComponentGuid}")
```
Comment (mandy-chessell) [inline open-metadata-implementation/access-services/governance-engine/governance-engine-spring/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/spring/GovernanceEngineOMASResource.java]: assetComponentGuid or assetGuid?

File: open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/objects/Context.java (hunk 1 above)
Comment (danielaotelea) [reply open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/objects/Context.java]: added

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/util/ExceptionHandler.java (hunk 4 above)
Comment (danielaotelea) [reply open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/util/ExceptionHandler.java]: added

File: open-metadata-implementation/access-services/governance-engine/governance-engine-spring/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/spring/GovernanceEngineOMASResource.java (hunk 5 above)
Comment (danielaotelea) [reply open-metadata-implementation/access-services/governance-engine/governance-engine-spring/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/spring/GovernanceEngineOMASResource.java]: assedGuid. Fixed in the last commit

File: open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/processor/ContextBuilder.java (hunk 2 above)
Comment (danielaotelea) [reply open-metadata-implementation/access-services/governance-engine/governance-engine-server/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/server/processor/ContextBuilder.java]: added

**Pull request merged**